packages/redux/src/selectors.ts

```diff
@@ -34,80 +34,82 @@ export const getFileStorage = <State extends RootState = RootState>(
 
 export const getWorkspaceStorage = <State extends RootState = RootState>(
   store: State
-): Storage['workspaceStorage'] => getStorage(store).workspaceStorage;
+): State['storage']['workspaceStorage'] => getStorage(store).workspaceStorage;
```

Comment:
These probably don't need to set a default generic value. It should be inferred from the param just fine

Reply:
So, should I remove the defaults from all similar selectors?